samples/ChangeFeedProcessor/DocumentDB.ChangeFeedProcessor.Test/Observer/TestObserver.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.Azure.Documents;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace DocumentDB.ChangeFeedProcessor.Test.Observer
+{
+    class TestObserver : IChangeFeedObserver
+    {
+        public TestObserver(IChangeFeedObserver parent)
+        {
+            Debug.Assert(parent != null);
+            this.parent = parent;
+        }
+
+        public Task OpenAsync(ChangeFeedObserverContext context)
+        {
+            return parent.OpenAsync(context);
+        }
+
+        public Task CloseAsync(ChangeFeedObserverContext context, ChangeFeedObserverCloseReason reason)
+        {
+            return parent.CloseAsync(context, reason);
+        }
+
+        public Task ProcessChangesAsync(ChangeFeedObserverContext context, IReadOnlyList<Document> docs)
+        {
+            return this.parent.ProcessChangesAsync(context, docs);
+        }
+
+        IChangeFeedObserver parent;
```

Comment:
nit: reorder on top of the constructor and possibly mark it as `private readonly`

Reply:
done